engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala

```diff
@@ -632,12 +633,17 @@ class IrToTruffle(
                         }
                   }
 
+                val funcSchema = FunctionSchema
```

Comment:
The builder is nice to use even from Scala. Btw. what does it mean `_*`?

Reply:
AFAIK, it is merely a type annotation for the Scala compiler to treat the parameter as vararg.